test/testenv/verificationutils.go

```diff
@@ -559,39 +559,70 @@ func VerifyPVCsPerDeployment(deployment *Deployment, testenvInstance *TestEnv, d
 	}
 }
 
-// VerifyAppInstall verify that app of specific version is installed
-func VerifyAppInstall(deployment *Deployment, testenvInstance *TestEnv, ns string, podName string, apps map[string]string, versionCheck bool, statusCheck string) {
-	for appName, version := range apps {
-		status, versionInstalled, err := GetPodAppStatus(deployment, podName, ns, appName)
-		logf.Log.Info("App info returned for app", "App-name", appName, "status", status, "versionInstalled", versionInstalled, "error", err)
-		gomega.Expect(err).To(gomega.Succeed(), "Unable to get app status on pod ")
-		gomega.Expect(strings.ToLower(status)).Should(gomega.Equal(strings.ToLower(statusCheck)))
-		if versionCheck {
-			gomega.Expect(versionInstalled).Should(gomega.Equal(version))
+// VerifyAppInstalled verify that app of specific version is installed
+func VerifyAppInstalled(deployment *Deployment, testenvInstance *TestEnv, ns string, pods []string, apps []string, versionCheck bool, statusCheck string, checkupdated bool, clusterWideInstall bool) {
+	for _, podName := range pods {
+		if !strings.Contains(podName, "monitoring-console") {
+			for _, app := range apps {
+				appName := AppInfo[app]["App-name"]
+				status, versionInstalled, err := GetPodAppStatus(deployment, podName, ns, appName)
+				logf.Log.Info("App info returned for app", "App-name", appName, "status", status, "versionInstalled", versionInstalled, "error", err)
+				gomega.Expect(err).To(gomega.Succeed(), "Unable to get app status on pod ")
+				comparsion := strings.EqualFold(status, statusCheck)
+				//Check the app is installed on specific pods and un-installed on others for cluster-wide install
+				if clusterWideInstall {
+					if strings.Contains(podName, "-indexer-") || strings.Contains(podName, "-search-head-") || strings.Contains(podName, "-standalone-") {
```

Comment:
Cluster wide does not contain `standalone`

Reply:
modified